sdks/python/apache_beam/runners/dataflow/dataflow_job_service.py

```diff
@@ -0,0 +1,92 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import logging
+import sys
+
+from apache_beam.options import pipeline_options
+from apache_beam.portability.api import endpoints_pb2
+from apache_beam.runners.dataflow import dataflow_runner
+from apache_beam.runners.job import utils as job_utils
+from apache_beam.runners.portability import local_job_service
+from apache_beam.runners.portability import local_job_service_main
+from apache_beam.runners.portability import portable_runner
+
+
+class DataflowBeamJob(local_job_service.BeamJob):
+  """A representation of a single Beam job to be run on the Dataflow runner.
```

Comment:
Looks like pydoc doesn't like this class inheriting from BeamJob - `py:class reference target not found: apache_beam.runners.portability.local_job_service.BeamJob`

I think its because portability is excluded from py_doc here - https://github.com/apache/beam/blob/bf289863c3871991b8832ab014d7305542f3640f/sdks/python/scripts/generate_pydoc.sh#L63

You should be able to fix it by adding `apache_beam.runners.portability.local_job_service.BeamJob` to the private class exclusion list - https://github.com/apache/beam/blob/bf289863c3871991b8832ab014d7305542f3640f/sdks/python/scripts/generate_pydoc.sh#L177